pom.xml

```diff
@@ -180,6 +180,20 @@
             </goals>
           </execution>
 
+          <execution>
+            <phase>package</phase>
+            <configuration>
+              <tasks>
+                <replace token= "&lt;value&gt;50&lt;/value&gt;" value="&lt;value&gt;0&lt;/value&gt;" dir="${distro.openmrsConfigDir}">                                 
+                  <include name="globalproperties/globalproperties_core-demo.xml"/>
+                </replace>
+              </tasks>
```

Comment:
Rather than this, let's try to override the GP through our own config file in FOSS, possibly with the same file name.

Reply:
I've addressed this with a PR on ozone-config: https://github.com/ozone-his/ozone-config/pull/30